docs/source/get_started/installation.rst

```diff
@@ -163,6 +199,55 @@ If you are using the Windows and the PowerShell:
     pytest -m "not slow" deepchem # optional
 
 
+From source lightweight guide
+-------------------------------------
+
+**Installing via these steps will ensure you are installing from the source**.
+
+**Prerequisite**
+
+- Shell: Bash, Zsh, PowerShell
+- Conda: >4.6
+
+
+First, please clone the deepchem repository from GitHub.
+
+.. code-block:: bash
+
+    git clone https://github.com/deepchem/deepchem.git
+    cd deepchem
+
+We would advise all users to use conda environment, following below-
+
+.. code-block:: bash
+
+    conda create --name deepchem python=3.7
```

Comment:
We can recommend [python 3.8.](https://github.com/deepchem/deepchem/pull/2592#discussion_r667406513). We should also update this in other parts of README.md (see first line in this [section](https://github.com/deepchem/deepchem/#requirements))

Reply:
Yeah sure, will do